zerver/lib/bugdown/__init__.py

```diff
@@ -410,6 +410,18 @@ def fetch_tweet_data(tweet_id: str) -> Optional[Dict[str, Any]]:
 META_START_RE = re.compile('^meta[ >]')
 META_END_RE = re.compile('^/meta[ >]')
 
+def get_youtube_urls(urls: Set[str]) -> Set[str]:
```

Comment:
I wonder if we should start calling this something like `get_whitelisted_domain_urls` since we might soon include `dropbox` URLs at least, here. And also given that we are calling it in `actions.py`, etc., a more generic name might make more sense.